tasks/scrapeData/runScraper.js

```diff
@@ -0,0 +1,131 @@
+import * as datetime from '../../lib/datetime.js';
+import * as geography from '../../lib/geography.js';
+
+const numericalValues = ['cases', 'tested', 'recovered', 'deaths', 'active'];
```

Comment:
I was thinking we should put this in a file somewhere, but it was late and I couldn't think of where it should live...

It's used in a few random places across the project (CSV generation, validation, etc).

Thoughts on a name? `lib/schema.js` sounds wrong

Reply:
I think it definitely needs to live somewhere, and `lib/schema.js` sounds like a good name

Reply:
Yep, schema or constants or something, good to move to a file 👍